classes/ProductSale.php

```diff
@@ -97,6 +97,12 @@ public static function getBestSales($idLang, $pageNumber = 0, $nbProducts = 10,
             $orderWay = 'DESC';
         }
 
+        if ($orderWay === 'random') {
```

Comment:
don't execute line above if orderWay is random :thinking:

Reply:
Hi @PierreRambaud you ask me for this lines ?

https://github.com/PrestaShop/PrestaShop/blob/develop/classes/ProductSale.php#L85-L89

Or if it's just above

https://github.com/PrestaShop/PrestaShop/blob/develop/classes/ProductSale.php#L95-L98

i added random for Validate::isOrderWay($orderWay);

Thank you

Reply:
I mean line 85 & 91 because orderBy remove $orderBy and $orderTable